assemble/bin/accumulo-cluster

```diff
@@ -202,6 +294,8 @@ function start_all() {
   done
 
   for group in $SSERVER_GROUPS; do
+    var_name="NUM_SSERVERS_${group}"
+    [[ -n ${!var_name} ]] && NUM_SSERVERS=${!var_name}
```

Comment:
The variable indirection also lends itself to possibly consolidating and reusing a lot of these loops in a function. Doesn't have to be done here... just an idea for subsequent improvement.

Reply:
created #4770 as a follow-on